.github/workflows/downstream-maven-plugins.yaml

```diff
@@ -31,3 +33,20 @@ jobs:
     - run: sudo apt-get update -y
     - run: sudo apt-get install libxml2-utils
     - run: .kokoro/client-library-check.sh ${{matrix.repo}} ${{matrix.job-type}}
+  javadoc-with-doclet:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        repo:
+        - java-bigtable
+    steps:
+    - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4
+    - uses: actions/setup-java@v3
+      with:
+        distribution: temurin
+        java-version: 11
+    - run: java -version
+    - run: sudo apt-get update -y
+    - run: sudo apt-get install libxml2-utils
+    - run: .kokoro/javadoc-generation-with-doclet.sh ${{matrix.repo}}
```

Comment:
Can you rename the script to `client-library-check-javadoc-with-doclet.sh`? I want the name to tell it's similar to client-library-check.sh.